eng/icu.proj

```diff
@@ -37,4 +37,22 @@
             Lines="@(_ItemsFromEmscriptenVersionFile->Replace('&quot;', ''))"
             Overwrite="true" />
   </Target>
+  <Target Name="PatchWasiSdk" Condition="'$(TargetOS)' == 'Wasi'">
+    <ReadLinesFromFile File="../.devcontainer/wasi-sdk-version.txt">
```

Comment:
Why is the version in `.devcontainer`? We should move it to a msbuild property in `eng/Versions.props`.

Reply:
Moved